### ISSUE TITLE
How to handle multi tensors input?

### ISSUE THREAD

**Issue by doubler** [tensorflow/serving#9]
The mnist example shows how to use the model with one input tensor x and one output tensor y.
While I have two input, tensor x and a tf.placeholder named dropout_prob.
If I don't specify the dropout_prob value, it will produce error output log like
```
W external/tf/tensorflow/core/common_runtime/executor.cc:1157] 0x7fcf1c0023c0 Compute status: Invalid argument: You must feed a value for placeholder tensor 'dropout_prob' with dtype float
	 [[Node: dropout_prob = Placeholder[dtype=DT_FLOAT, shape=[], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]
```

Would you please show some example with multi input tensors or multi output tensors?

**Comment by doubler** [comment]
I have solved my problem by pre-define dropout_prob as a tf.constant before export the model.
While I'm still looking forward the multi input and output example. :)

**Comment by fangweili** [comment]
Unfortunately, neither our standard ClassificationSignature nor RegressionSignature supports multiple input binding for now. You can however, use GenericSignature to specify arbitrary number of inputs and outputs.

You can use exporter.generic_signature() to create it. Then provide it as default_signature in exporter.export().

**Comment by doubler** [comment]
Got it. Thanks for explanation.

**Comment by insectatorious** [comment]
@doubler would it be possible for you to share a code example of how you set the `dropout_prob` using `tf.constant`? I'm confused about how to fit this into the export step.

**Comment by doubler** [comment]
@insectatorious no need to set the `dropout_prob`, just feed it with 1.0 as input when serving online

**Comment by fabienbaradel** [comment]
@doubler I don't understand your explanation. What do you mean by "feed it with 1.0 as input"?

Below is the inception signature for a standard image classification:
```
      classification_signature = (
          tf.saved_model.signature_def_utils.build_signature_def(
              inputs={
                  tf.saved_model.signature_constants.CLASSIFY_INPUTS:
                      classify_inputs_tensor_info
              },
              outputs={
                  tf.saved_model.signature_constants.CLASSIFY_OUTPUT_CLASSES:
                      classes_output_tensor_info,
                  tf.saved_model.signature_constants.CLASSIFY_OUTPUT_SCORES:
                      scores_output_tensor_info
              },
              method_name=tf.saved_model.signature_constants.
              CLASSIFY_METHOD_NAME))
```

Could you show how is yours with two inputs? Do you have to redefine the `tf.saved_mode.signature_constants`?

Thanks for help.
